test/export/test_torchbind.py

```diff
@@ -182,10 +182,12 @@ def forward(self, x, n):
         self.assertExpectedInline(
             ep.graph_module.code.strip(),
             """\
-def forward(self, obj_attr, x, n):
-    call_torchbind = torch.ops.higher_order.call_torchbind(obj_attr, 'add_tensor', x);  obj_attr = None
-    add = torch.ops.aten.add.Tensor(x, call_torchbind);  x = call_torchbind = None
-    return (add,)""",
+def forward(self, token, obj_attr, x, n):
+    with_effects = torch._higher_order_ops.effects.with_effects(token, torch.ops._higher_order_ops.call_torchbind, obj_attr, 'add_tensor', x);  token = obj_attr = None
+    getitem = with_effects[0]
+    getitem_1 = with_effects[1];  with_effects = None
+    add = torch.ops.aten.add.Tensor(x, getitem_1);  x = getitem_1 = None
+    return (getitem, add)""",  # noqa: B950
```

Comment:
Why are we returning more outputs now?

Reply:
it's the token! this is the graph from the ExportedProgram's functional aten graph so it contains the token, but when you run it eagerly, ExportedProgram has post-processing so that it doesn't become a user output.